script/utils.js

```diff
@@ -1,9 +1,13 @@
 const { spawn, spawnSync } = require('child_process');
 
-const runCommand = (cmd, forcedExitCode = null) => {
+const runCommand = cmd => {
   const child = spawn(cmd, [], { shell: true, stdio: 'inherit' });
+
+  // If the child process exits abnormally, exit parent with the same code
   child.on('exit', code => {
-    process.exit(forcedExitCode === null ? code : forcedExitCode);
+    if (code) {
+      process.exit(code);
```

Comment:
Would it be safer to use `process.exitCode` here? See https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-process-exit.md

Reply:
Thanks, that makes total sense, so I tried this:
```
child.on('exit', code => {
  if (code) {
    process.exitCode = code;
    throw new Error(`"${cmd}" failed with exit code ${code}`);
  }
});
```
However in my testing, the parent process exits with code 1 regardless of the value of `code` (and therefore of `process.exitCode`). It seems like this should work based on the node docs. Maybe there's something else that's overriding the value of `process.exitCode`. But a search for that text didn't come up with anything. Any ideas about it?